Resources/Maps/meta.yml

```diff
@@ -54,6 +54,7 @@ entities:
     - type: MetaData
     - type: Transform
     - type: Map
+      mapPaused: True
```

Comment:
Isn't this bad?

Reply:
https://github.com/space-wizards/space-station-14/pull/28362#issuecomment-2142925923

Why was ``mapPaused: True`` an issue for the nukie base but not this?

Reply:
it isnt a station